gm4_heart_canisters/data/gm4/advancements/heart_canisters_tier_1.json

```diff
@@ -0,0 +1,27 @@
+{
+  "display":{
+    "icon":{
+      "item":"player_head",
+      "nbt":"{SkullOwner:{Id:\"25b1ff93-7926-1cd7-53fe-8e6050792daa\",Properties:{textures:[{Value:\"eyJ0aW1lc3RhbXAiOjE0MjkzNDc5MDU4NzMsInByb2ZpbGVJZCI6IjYzY2JkZjhkNDg4OTQ3NWY5NDQxMjk3ZTRhM2Q1NjczIiwicHJvZmlsZU5hbWUiOiJWZWxlVCIsImlzUHVibGljIjp0cnVlLCJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNTNjZjA2ZDViNDdiNzQ2ZDE2ZWU2MzExODdjYjg1N2Q4YzE2YmZjZDU3MGE0MGYyODZiMmMzODg2N2NmZGEifX19\"}]}},display:{Name:\"heart_canisters logo 1\"}}"
+    },
+    "title":"Can't Buy Me Love",
+    "description":{
+      "text":"Craft a tier 1 Heart Canister.",
+      "color":"gray"
+    }
+  },
+  "parent":"gm4:root",
+  "criteria":{
+    "craft_tier_1":{
+      "trigger":"minecraft:inventory_changed",
```

Comment:
This will also trigger on the player getting a heart can from another player, not just crafting it themselves. Since most vanilla crafting achievements also can be earned in this manner, it should be alright to leave it this way, even though the title is "Craft a heart can"